src/platforms/javascript/common/configuration/tree-shaking/index.mdx

```diff
@@ -36,7 +36,49 @@ To make optional code eligible for tree shaking, you can replace various flags i
 : Replacing this flag with `false` will tree shake all code in the SDK that is related to performance monitoring.
 **Attention:** `__SENTRY_TRACING__` must not be replaced with `false` when you're using any performance monitoring-related SDK features (e.g. `Sentry.startTransaction()`). This flag is intended to be used in combination with packages like `@sentry/next` or `@sentry/sveltekit`, which automatically include performance monitoring functionality.
 
-<PlatformSection notSupported={["javascript.nextjs"]}>
+`__RRWEB_EXCLUDE_CANVAS__`
+
+: Replacing this flag with `true` will tree shake all code in the SDK that is related capturing canvas elements with Session Replay. This is only relevant when using [Session Replay](../../session-replay/). You can enable this flag if you don't have any canvas elements on your page you care to record.
+
+`__RRWEB_EXCLUDE_IFRAME__`
+
+: Replacing this flag with `true` will tree shake all code in the SDK that is related capturing iframe content with Session Replay. This is only relevant when using [Session Replay](../../session-replay/). You can enable this flag if you don't have any iframes on your page you care to record.
```

Comment:
I realize the following ones also start with 'replace with true' where my suggestion above starts by clarifying this is relevant to Session Replay.

How about being clear at the top under a header 
```
### Session Replay

The following options are specific for Session Replay users. If you're not using Session Replay, (by adding `new Replay()` integration) these will not take any effect in your final bundle size
```

then we can skip being specific about it in each entry

Reply:
hmm not 100% sure, then we need to basically reorganize all of this, as for consistency then we should have a section (?) for performance (`__SENTRY_TRACE__`) as well 🤔 There would be a bunch of headings then, e.g.:

## List of Flags

### Common Flags

`__SENTRY_DEBUG__`: 

....

### Performance

`__SENTRY_TRACE__`: 

...

### Session Replay

`__RRWEB_EXCLUDE_CANVAS__`: 

...

`__RRWEB_EXCLUDE_IFRAME__`: 

...

`__RRWEB_EXCLUDE_SHADOW_DOM__`: 

...


--> Do we like this better?

Reply:
I don't think this matters too much overall but if we go with this categorization, `__SENTRY_DEBUG__` and `__SENTRY_TRACING__` should both be under "common" because they remove bytes from the core SDK, while the Replay flags are only relevant when using replay.